source/administration/config-settings.rst

```diff
@@ -2,12 +2,14 @@ Configuration Settings
 ======================
 
 .. note::
-   The order of the configuration settings below are reflective of a reorganization of the System Console in v5.12 released on June 16th, 2019. To view the configuration settings based on the organization of the System Console in versions prior to v5.12, please see `this documentation <https://docs.mattermost.com/administration/prev-config-settings.html>`_ instead.
+   The order of the configuration settings below are reflective of a reorganization of the System Console in version 5.12 released on June 16th, 2019. To view the configuration settings based on the organization of the System Console in versions prior to version 5.12, please see `this documentation <https://docs.mattermost.com/administration/prev-config-settings.html>`_ instead.
 
 Mattermost configuration settings are maintained in the configuration file ``config.json``, located in the ``mattermost/config`` directory. You can modify the configuration file using the System Console, or by using a text editor to modify it directly.
 
 The default location of ``config.json`` is in the ``mattermost/config`` directory. Mattermost must have write permissions to ``config.json``, otherwise changes made in the System Console will have no effect.
 
+On new installations starting in version 5.14, the default.json file used to create the initial config.json has been removed from the binary and replaced it with a build step that generates a fresh config.json.  This is to ensure the initial configuration file has all with correct defaults provided in the server code. 
```

Comment:
I can understand this sentence because I have enough context, but propose we share with someone less familiar, e.g. someone in support, to see if it's obvious to them?

Reply:
Had Sven review - he thought the paragraph was good to have included.  We added another sentence that existing ``config.json`` files are not affected.

Reply:
thanks Katie!